src/Engine/XYCoord.java

```diff
@@ -13,6 +15,10 @@ public XYCoord(int x, int y)
     xCoord = x;
     yCoord = y;
   }
+  public XYCoord(Unit u)
```

Comment:
Interesting. Why this instead of `unit.getCoord()`?

Reply:
No reason, really.